### DISCUSSION TITLE
feat(Table): Fuzzy search allow Ignore case

### DISCUSSION THREAD

**Discussion by ElanCao** [dotnetcore/BootstrapBlazor#4515] [Q&A]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Is your feature request related to a problem? Please describe the problem.

Currently Table fuzzy search is case sensitive.

### Describe the solution you'd like

Add one option or default, allow Ordinal_Ignore_Case.

**Comment by bb-auto** [comment]
@ElanCao Thank you for contacting us. We will give feedback later.

**Comment by ArgoZhang** [comment]
@ElanCao 感谢提交 Issue 请帮忙点一下项目右上角 star 谢谢啦
